frontend/src/pages/ProjectDetails.tsx

```diff
@@ -92,11 +93,26 @@ const ProjectDetailsPage = () => {
     },
   ]
   const projectStats = [
-    { icon: faUsers, value: `${project?.contributors_count || 'No'} Contributors` },
-    { icon: faCodeFork, value: `${project?.forks_count || 'No'} Forks` },
-    { icon: faStar, value: `${project?.stars_count || 'No'} Stars` },
-    { icon: faCode, value: `${project?.repositories_count || 'No'} Repositories` },
-    { icon: faExclamationCircle, value: `${project?.issues_count || 'No'} Issues` },
+    {
+      icon: faUsers,
+      value: `${project?.contributors_count || 'No'} Contributor${pluralize(project.contributors_count, 's')}`,
+    },
+    {
+      icon: faCodeFork,
+      value: `${project?.forks_count || 'No'} Fork${pluralize(project.forks_count, 's')}`,
```

Comment:
Is there a way to use `s` as a default when it's not passed? It's almost always `-s` prefix.